### PR TITLE
[Refactoring] - reworked old tests + cleanup

### PR DESCRIPTION
**Pull request by mmajerni**

### CODE REVIEW

File: src/test/java/apicurito/tests/utils/slenide/OperationUtils.java [hunk 1]
```diff
@@ -25,11 +24,10 @@ public static SelenideElement getOperationRoot() {
 
     public static void setResponseStatusCode(Integer code) {
         log.info("Setting status code to {}", code);
+        SelenideElement dropdownElement = getOperationRoot().$("#addResponseModal").$(By.className("dropdown"));
```
Comment (mcada) [inline src/test/java/apicurito/tests/utils/slenide/OperationUtils.java]: Add static element class with used selectors.

File: src/test/java/apicurito/tests/utils/slenide/OperationUtils.java [hunk 2]
```diff
@@ -25,11 +24,10 @@ public static SelenideElement getOperationRoot() {
...
 
-        getOperationRoot().$("#statusCodeMenu").click();
-
-        getOperationRoot().$$("ul").filter(attribute("class", "dropdown-menu")).first()
-                .$$("a").filter(text(code.toString())).first().click();
+        dropdownElement.$("#statusCodeDropDown").click();
```
Comment (mcada) [inline src/test/java/apicurito/tests/utils/slenide/OperationUtils.java]: Add static element class with used selectors.

File: src/test/resources/features/dataTypesTests.feature [hunk 3]
```diff
@@ -114,4 +125,31 @@ Feature: Test everything on data types page
...
+    And check that property "age" is "Not Required"
+
+    And check that data type property "isMan" is created
+    And check that type is "Boolean" for property "isMan"
+    And check that property "isMan" is "Not Required"
+
+    And check that data type property "name" is created
+    And check that type is "String" for property "name"
+    And check that type as is "String" for property "name"
+    And check that property "name" is "Not Required"
```
Comment (mcada) [inline src/test/resources/features/dataTypesTests.feature]: missing eof new line

File: src/test/resources/features/mainPageTests.feature [hunk 4]
```diff
@@ -1,16 +1,129 @@
...
+    Then check that API description is "New API desc"
+
+  @setAPIconsumes
+  Scenario: set API consumes
+    When create a new API
+    And set consumes to "text/xml"
+    And save API as "json" and close editor
+    And import API "tmp/download/openapi-spec.json"
+    Then check that API consume "text/xmlapplication/json"
+    #TODO add space and check every label separately
```
Comment (mcada) [inline src/test/resources/features/mainPageTests.feature]: do you have jira task for this todo?

File: src/test/resources/features/mainPageTests.feature [hunk 5]
```diff
@@ -1,16 +1,129 @@
...
+
+  @setAPIproduces
+  Scenario: set API produces
+    When create a new API
+    And set produces to "text/xml"
+    And save API as "json" and close editor
+    And import API "tmp/download/openapi-spec.json"
+    Then check that API produce "text/xmlapplication/json"
+    #TODO add space and check every label separately
```
Comment (mcada) [inline src/test/resources/features/mainPageTests.feature]: do you have jira task for this todo?

File: src/test/resources/features/mainPageTests.feature [hunk 6]
```diff
@@ -1,16 +1,129 @@
...
+      | api   |
+      | basic |
+
+    And save API as "json" and close editor
+    And import API "tmp/download/openapi-spec.json"
+
+    Then check that API security requirement "api, basic" exists
+
+  @mainPage
   Scenario: Test the api main page (set everything expect security)
```
Comment (mcada) [inline src/test/resources/features/mainPageTests.feature]: typo in `expect`

File: src/test/resources/features/mainPageTests.feature [hunk 7]
```diff
@@ -69,55 +181,10 @@ Feature: Test everything on main page
...
-
-#    And check oauth security scheme   //TODO cannot be done now (//TODO check in Source tab???)
-#      | OAuth2name     | OAuth2 description | Access Code | https://github.com/Apicurio/ | https://github.com/Apicurio/apicurito |
-
-    Then check all for errors
+  Scenario: #kebab menu for paths and data types
 
-  Scenario: #kebab menu for paths and data types
\ No newline at end of file
+  #TODO after closing:
```
Comment (mcada) [inline src/test/resources/features/mainPageTests.feature]: do you have jira task for this todo?

File: src/test/resources/features/mainPageTests.feature [hunk 8]
```diff
@@ -69,55 +181,10 @@ Feature: Test everything on main page
...
+  # https://github.com/Apicurio/apicurio-studio/issues/656
+  # https://github.com/Apicurio/apicurio-studio/issues/657
+  #  And add scopes to scheme "OAuth2name"
+  #  And add scopes to security requirement "OAuth2name, APIkeySecurityName" and OAuth scheme "OAuth2name"
```
Comment (mcada) [inline src/test/resources/features/mainPageTests.feature]: missing eof new line

File: src/test/resources/features/pathTests.feature [hunk 9]
```diff
@@ -267,4 +267,122 @@ Feature: Path tests
     And check that overridden path parameter "id" has description "override parameter id desc"
 
     #TODO Add test -> create path parameter in operation
-    #TODO Add tests -> create path QP and HP
\ No newline at end of file
+    #TODO Add tests -> create path QP and HP
```
Comment (mcada) [inline src/test/resources/features/pathTests.feature]: do you have jira task for this todo?

File: src/test/resources/features/pathTests.feature [hunk 10]
```diff
@@ -267,4 +267,122 @@ Feature: Path tests
...
+
+    And create path parameter "email"
+    And set description "parameter email desc" for path parameter "email"
+    And set path parameter type "String" for path parameter "email"
+    And set path parameter type as "Byte" for path parameter "email"
+
+    And select operation "GET"
+    And override parameter "email"
+    And set description "override parameter email desc" for override path parameter "email" in operation
+    #TODO create parameter in operation
```
Comment (mcada) [inline src/test/resources/features/pathTests.feature]: do you have jira task for this todo?

**Comment by mmajerni** [comment]
@mcada added static element, eof new lines and corrected typo.

**Pull request merged**